ssm/helpers.go

```diff
@@ -54,7 +54,7 @@ func addInstanceInfo(instanceID *string, tags []ec2helpers.InstanceTags, instanc
 }
 
 // RunInvocations invokes an SSM document with given parameters on the provided slice of instances
-func RunInvocations(sp *session.Pool, sess *ssm.SSM, instances []*ssm.InstanceInformation, params *invocation.RunShellScriptParameters, dryRun bool, resultsPool *invocation.ResultSafe) (err error) {
+func RunInvocations(sp *session.Pool, sess *ssm.SSM, instances []*ssm.InstanceInformation, params *invocation.RunShellScriptParameters, dryRun bool, maxConcurrency string, maxErrors string, resultsPool *invocation.ResultSafe) (err error) {
```

Comment:
More an observation than a hard change, but we may want to think on abstracting these flags behind some other config structure at some point as I imagine this will keep bloating over time.